src/test/functional/com/MyApp/budgetControl/domain/ServiceOrchestratorTest.java

```diff
@@ -0,0 +1,67 @@
+package com.MyApp.budgetControl.domain;
+
+import com.MyApp.budgetControl.domain.expense.ExpenseRequestDTO;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.jayway.jsonpath.JsonPath;
+import lombok.SneakyThrows;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+
+@SpringBootTest
+@AutoConfigureMockMvc
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+public class ServiceOrchestratorTest {
```

Comment:
Are You sure You need it? Isnt this just duplication of tests that You do in UserController/ExpenseControler/CategoryControler? You are QA, think about why You need those tests, what are You testing? Maybe You can just write unit test?

Reply:
I wanted to test if the new expenses were added to the user repository, now that I have moved this logic to the ExpenseService layer, I have added a unit test in ExpenseServiceTest to cover this case. 
And yes without that it is just a test duplication.